src/main/java/org/skriptlang/skriptworldguard/elements/expressions/ExprRegionFlag.java

```diff
@@ -0,0 +1,118 @@
+    public boolean init(Expression<?>[] expression, int matchedPattern, Kleenean kleenean, ParseResult parseResult) {
+        exprFlag = (Expression<String>) expression[0];
+        exprRegions = (Expression<WorldGuardRegion[]>) expression[1];
+        return true;
+    }
+
+    @Override
+    protected Flag[] get(Event event, WorldGuardRegion[] region) {
+        Flag<?> flag = Flags.fuzzyMatchFlag(wg.getFlagRegistry(), exprFlag.getSingle(event));
+        List<Flag> result = new ArrayList<>();
+        for (WorldGuardRegion wrg : region){
+            ProtectedRegion rg = wrg.getRegion();
+            if (rg != null){
+                result.add((Flag) rg.getFlag(flag));
+            }
+        }
+        return (Flag[]) result.toArray();
+    }
+
+    @Override
+    public Class<?>[] acceptChange(ChangeMode mode){
+        switch (mode){
+            case SET:
+            case DELETE:
+                return CollectionUtils.array(String.class);
+        }
+        return null;
+    }
+
+    public void change(Event event, Object[] delta, Changer.ChangeMode mode){
+        Flag<?> flag;
+        if(exprFlag.getSingle(event) == null || exprRegions.getSingle(event) == null){
+            return;
+        }
+        WorldGuard wg = WorldGuard.getInstance();
+        WorldGuardRegion[] regions = getExpr().getArray(event);
+        flag = Flags.fuzzyMatchFlag(wg.getFlagRegistry(), exprFlag.getSingle(event));
+        for (WorldGuardRegion region : regions){
+            ProtectedRegion rg = region.getRegion();
+            if (rg != null && flag != null){
+                switch(mode){
+                    case SET:
+                        if(delta != null){
+                            if (delta[0] instanceof Boolean) {
+                                if ((Boolean) delta[0]) {
+                                    rg.setFlag((StateFlag) flag, StateFlag.State.ALLOW);
+                                } else {
+                                    rg.setFlag((StateFlag) flag, StateFlag.State.DENY);
+                                }
+                            }
+                            else if (delta[0] instanceof String) {
+                                rg.setFlag((StringFlag) flag, (String) delta[0]);
+                            } else if (delta[0] instanceof Integer) {
+                                rg.setFlag((IntegerFlag) flag, (int) delta[0]);
+                            } else if (delta[0] instanceof Double) {
+                                rg.setFlag((DoubleFlag) flag, (double) delta[0]);
+                            } else {
+                                SkriptWorldGuard.getInstance().getLogger().warning("Region flag " + flag.getName() + " cannot be set to: " + delta[0]);
+                            }
+                        }
+                    case DELETE:
+                        rg.setFlag(flag, null);
+                    default:
+                        Skript.error("A flag can only be set or cleared.");

```

Comment:
```suggestion
                        assert false;

```
I think it's okay to assert false instead